### ISSUE TITLE
Debugging instructions are missing

### ISSUE THREAD

**Issue by vitalyPru** [exercism/exercism#6262]
Please, consider the possibility to allow debug print during Rst tests.

**Comment by kotp** [comment]
This is already possible.  I do not remember how to expose this though.

But do you not have this ability when you run your tests locally?

**Comment by vitalyPru** [comment]
I just use only cloud version. 
And if I put println!() in my solution, there are no any output in test results

**Comment by kotp** [comment]
Yes, `println()` would not be expected to show in the reports for testing.

If getting serious about the language, I do encourage local development though.  It gives control back to you, rather than relying on what we provide on the server.

**Comment by kotp** [comment]
I have sent out a "do you remember how" question in our chat.  Hopefully an answer comes soon for you regarding debugging output in the tests.

**Comment by vitalyPru** [comment]
I found,, that debug output works for structs, which derived Debug trait.
So. issue is closed.

**Comment by iHiD** [comment]
Does this need adding to the docs for this? @ErikSchierboom - there's a md doc for this specifically, right?

**Comment by SleeplessByte** [comment]
@iHiD correct, this section is missing for rust because the content is missing

![image](https://user-images.githubusercontent.com/1964376/156797992-b9ebeabe-1b41-4c0c-99ea-993218078908.png)
